src/Counters.hpp

```diff
@@ -30,6 +30,7 @@ class Counters
 {
 public:
   Counters();
+  Counters(bool from_log);
```

Comment:
Suggestion: Add a `bool from_log` parameter to `Statistics::format_human_readable` instead. Then we can let `Counters` remain unaware of its origin.